src/index.js

```diff
@@ -48,11 +48,16 @@ function format(options) {
     text = getTextFromFilePath(filePath),
     eslintPath = getModulePath(filePath, 'eslint'),
     prettierPath = getModulePath(filePath, 'prettier'),
-    prettierOptions,
     prettierLast,
     fallbackPrettierOptions,
   } = options
 
```

Comment:
Maybe we can `const prettier = getPrettier(prettierPath);` here and pass `prettier` instead of `prettierPath` to `resolvePrettierOptions` and `createPrettify`?